index.js

```diff
@@ -8,6 +8,8 @@ var dist_dir = process.env.DIST || __dirname + '/dist/release/';
 
 module.exports = function (options) {
   // Options
+  var self = this;
```

Comment:
Could you remove `self` and rather just use `this`. Everything seems to be top level so self isn't needed. If you need to use `this` in another function can you make it a `() =>` function and then this will work as well.